src/Command/BackupCommand.php

```diff
@@ -112,23 +114,18 @@ private function doBackup(): string
         $this->backupper = $this->backupperFactory->create($this->connectionName);
         $filename = $this->storage->generateFilename($this->connectionName, $this->backupper->getExtension());
 
-        $this->backupper
+        $this
+            ->backupper
             ->setDestination($filename)
-            ->setVerbose($this->io->isVerbose())
             ->setExcludedTables($this->excludedTables[$this->connectionName] ?? [])
             ->setExtraOptions($this->extraOptions)
             ->ignoreDefaultOptions($this->ignoreDefaultOptions)
-            ->startBackup()
+            ->setVerbose($this->io->isVerbose())
```

Comment:
Same, I'm not sure verbosity is something the backuper should know, simply let it log everything, using logger levels.

Reply:
I close this discussion which is a duplicate.